packages/typespec-go/test/cadlranch/payload/multipartgroup/formdata_client_test.go

```diff
@@ -102,43 +102,12 @@ func TestFormDataClient_Complex(t *testing.T) {
 				Body: streaming.NopCloser(bytes.NewReader(pngFile)),
 			},
 		},
-		PreviousAddresses: []multipartgroup.Address{
-			{
-				City: to.Ptr("Y"),
-			},
-			{
-				City: to.Ptr("Z"),
-			},
-		},
 		ProfileImage: streaming.MultipartContent{Body: jpgFile},
 	}, nil)
 	require.NoError(t, err)
 	require.Zero(t, resp)
 }
 
-func TestFormDataClient_JSONArrayParts(t *testing.T) {
```

Comment:
Why was this removed?

Reply:
This tests have been removed from cadl-ranch new version and some other tests added. I could open another pr to add all added tests.